dexter/src/main/java/com/karumi/dexter/AndroidPermissionService.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2015 Karumi.
+ */
+
+package com.karumi.dexter;
+
+import android.app.Activity;
+import android.support.annotation.NonNull;
+import android.support.v4.app.ActivityCompat;
+import android.support.v4.content.ContextCompat;
+
+/**
+ * Wrapper class for all the static calls to the Android permission system
+ */
+class AndroidPermissionService {
```

Comment:
It won't hurt but I'll change its methods visibility to ``default`` to match the class